src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -847,7 +864,13 @@ public class TransactionData
         public Money Amount { get; set; }
 
         /// <summary>
-        /// A value indicating whether this is a coin stake transaction or not.
+        /// A value indicating whether this is a coinbase transaction or not.
+        /// </summary>
+        [JsonProperty(PropertyName = "isCoinBase", NullValueHandling = NullValueHandling.Ignore)]
+        public bool? IsCoinBase { get; set; }
```

Comment:
I also want to know how this can null? A transaction is either a coinbase tx or its not?

Reply:
The IsCoinStake JSON property is nullable, so I am merely matching that with the introduction of IsCoinBase. This was presumably originally done so that the wallet JSON does not get numerous unnecessary boolean fields (as coinbase/stake transactions are the exception rather than the rule).

Reply:
It would be probably better to replace it for the Coinbase as well instead.